fili-sql/src/main/java/com/yahoo/bard/webservice/sql/DruidQueryToSqlConverter.java

```diff
@@ -929,6 +941,32 @@ private void sqlAggregationHelper(
         return allGroupBys;
     }
 
+    /**
+     * Removes the quotes around the Sketch metric literal.
+     *
+     * @param sqlQuery  the sql string built by the RelBuilder.
+     *
+     * @return the sql string without quote around the sketch metric expression.
+     */
+    private String formatSketchQuery(String sqlQuery) {
```

Comment:
We were trying to avoid using string hacks to form queries since the string hack from intern code has created enough problems. 
I have told Ashish but I am not sure if he had synced up with you that the `RelBuilder.literal()` or some other functions in RelBuilder class could be used to create the function call sql code.

Reply:
I'm already using `RelBuilder.literal()` https://github.com/yahoo/fili/pull/1133/files#diff-564e0da92e321668ecc97c6ff955f3a03f326ad9daa6167fa11210d5503faac0R429-R433 
The intended use for literal is to create constants that's why there are quotes added that need to be removed in the end. Looking at the documentation I haven't found a better way to achieve that.